app/controllers/api/v1/countries_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Api::V1
+  class CountriesController < Api::V1::ApiController
+
+    load_and_authorize_resource :country, parent: false
+
+    def create
+      if country_record.save
+        render json: {}, status: 201
+      else
+        render json: @country.errors.to_json, status: 422
+      end
+    end
+
+    private
+
+    def country_record
+      @country = Country.where(stockit_id: country_params[:stockit_id]).first_or_initialize
```

Comment:
@swatijadhav similar to the other day, I'd like these controllers to continue to follow the conventions of assuming params[:id] first. That way it will be easy to add a country controller to GoodCity ember project and expect it to behave as any normal controller.

Can we do something like 

    def country_record
      if params[:id].present?
        @country = Country.where(id: params[:id]).first_or_initialize
      elsif country_params[:stockit_id].present?
        @country = Country.where(stockit_id: country_params[:stocki_id]).first_or_initialize
      else
        raise ActiveRecord::NotFound
      end
    end